src/components/wallet/earn/Validate/AddValidator.vue

```diff
@@ -193,8 +196,8 @@ export default class AddValidator extends Vue {
     startDate: string = new Date(Date.now() + MIN_MS * 15).toISOString()
```

Comment:
Wouldn't it be better to use self-explanatory constants instead of magic numbers?

Reply:
Not in the scope of this MR